deepchem/models/losses.py

```diff
@@ -976,6 +976,40 @@ def loss(node_emb, inputs):
         return loss
 
 
```

Comment:
Can you add the new losses to `losses.rst`?

Reply:
done